atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbMetricNames.java

```diff
@@ -33,6 +33,9 @@ private CellFilterMetrics() {
         public static final String EMPTY_VALUE = "emptyValuesCellFilterCount";
     }
 
+    public static final String LIBRARY_ORIGIN_TAG = "libraryOrigin";
```

Comment:
If we're going to ask datadog to make a new report itemizing costs, we need to make sure we use a consistent tag across all libraries. In the metric-schema PR, we're intending to use `libraryName`, so I don't think it will help much if you pick a different one

Reply:
My main worry here would be how metric registries handle duplicate tags - we are in the process of rolling out metric-schema as we touch new code. Do we know if that's okay? If we do then I'm happy to change this

Reply:
A read of `ExtraEntrySortedMap` in tritium suggests this might become a problem if the underlying metrics already have a `libraryName`.

Reply:
Ok - so it was just flagged to me that this is non-ideal for our metrics concerns - for dan's original comment. @iamdanfox is there any chance you could work with jeremy to figure out an agreeable solution? I guess in true Highlander style There Can Only Be One which makes me hackily think of an extra layer of indirection (you have a libraryOrigin -> atlasdb tag, and then that wraps anotehr metric set which _eitehr_ adds library name atlasdb or if already set, lets it still be set.

Reply:
I think a reasonable solution here could be to just make ExtraEntrySortedMap not freak out if you're asking to add a `libraryName:atlasdb` but the map already contains `libraryName:atlasdb`. Collisions should only a problem with same key and different value, but we currently barf on _all_ collisions...

Reply:
The intention I had when doing this was a bit different: when I wrote this I didn't really care about `libraryOrigin: atlasdb`, I just need a tag that won't be shared to be added here so that I can process the TaggedMetricSets nicely and then register them to the world outside AtlasDB.

The cost tracking was intended to be a separate thing. Can we be sure that the Atlas metric registries will never register any metrics that already have metricName? (e.g. dialogue, tritium?) - if so then I'm happy to change this to be libraryName.